extensions/parquet/table/plugin/channelprovider/s3/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
```

Comment:
Please share your suggestions on the names of these plugins and where exactly to place them